ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -160,377 +129,574 @@ function SignatureIntegrationForm({
     return (
         <>
             <PageSection variant="light" isFilled hasOverflowScroll>
-                <FormMessage message={message} />
-                <Form isWidthLimited>
-                    <FormikProvider value={formik}>
-                        <FormLabelGroup
-                            isRequired
-                            label="Integration name"
-                            fieldId="name"
-                            touched={touched}
-                            errors={errors}
-                        >
-                            <TextInput
+                <Flex direction={{ default: 'column' }}>
+                    <FlexItem>
+                        <Text>
+                            Image signatures are verified by ensuring that their signature has been
+                            signed by a trusted image signer. Configure at least one trusted image
+                            signer by specifying a Cosign public encryption key or a Cosign
+                            certificate chain. Multiple image signers may be combined in a single
+                            signature integration.
+                        </Text>
+                    </FlexItem>
+                    <FlexItem>
+                        <Text>
+                            Certificates that are contained in the image signature must not be
+                            expired. Communication with the transparency log Rekor is not supported.
+                        </Text>
+                    </FlexItem>
+                    <FlexItem>
+                        <Text>
+                            For more information, see{' '}
+                            <ExternalLink>
+                                <a
+                                    href={getVersionedDocs(
+                                        version,
+                                        'operating/verify-image-signatures'
+                                    )}
+                                    target="_blank"
+                                    rel="noopener noreferrer"
+                                >
+                                    RHACS documentation
+                                </a>
+                            </ExternalLink>
+                        </Text>
+                    </FlexItem>
+                    <FlexItem></FlexItem>
```

Comment:
The stray `<FlexItem></FlexItem>` and number of changed lines caused me to offer an alternative for your consideration:

![Alert_info_Verifying_image_signatures](https://github.com/user-attachments/assets/2abe341f-0cbc-4179-8b1d-c971a0756d7d)

If you like, it here is the recipe:
* Limit `Flex` to the added content. That is move up end tag and `Form` has original indent level.
* Enclose `Flex` in `Alert` element:

```tsx
                <Alert
                    title="Verifying image signatures"
                    component="p"
                    variant="info"
                    isInline
                    className="pf-v5-u-mb-lg"
                >
                    /* Flex and FlexItems here */
                </Alert>
```